src/components/FilterDropdown.jsx

```diff
@@ -16,15 +18,18 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 function FilterDropdown(props) {
-  const { label, items, defaultValue, removeNone, onChange } = props;
-  const [choice, setChoice] = useState(defaultValue);
+  const { label, items, value, removeNone, onChange } = props;
   const classes = useStyles();
+  const location = useLocation();
+  const params = queryString.parse(location.search, { parseNumbers: true });
+  const sortByParams =
+    params.sortBy === undefined ? 'deadlineAsc' : params.sortBy;
+  const [choice, setChoice] = useState(value === 'sorting' ? sortByParams : '');
```

Comment:
I'm not understanding why this is necessary. If the parent component passes what `value` should be and updates it whenever it needs to be something else, why set the choice to anything else? Using `value` also removes the need for the `choice` state variable.

Reply:
I think what you mean is just pass the `sortBy` from Scholarships and that will determine what is used. But again when prune is used it is undefined or something because it does not read in anything if u see

Reply:
Ah ok nvm I figured it out, nice you were completely correct the choice state is no longer needed. Cool thank you